docs/source/share_kedro_viz.md

```diff
@@ -109,3 +112,17 @@ You can control who can view your visualisation using [bucket and user policies]
 You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
 
 See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 
+
+## Platform-agnostic sharing with Kedro-Viz 
+
+In Kedro-Viz version 7.1.0, we introduced the `kedro viz build` command to enable you to publish and share Kedro-Viz to any static website hosting platform. Running this command from the command line interface (CLI) creates a `build` folder within the Kedro project, which contains a static Kedro-Viz app package ready for to publish.
```

Comment:
command to enable you to publish and share Kedro-Viz to any static website hosting platform. -> command **that enables** you to publish and share Kedro-Viz **on** any static website hosting platform.